lib/currency-locales/locales/ja.yml

```diff
@@ -1,26 +1,51 @@
+---
 ja:
   number:
     currency:
       format:
         delimiter: ","
-        format: "%u%n"
+        format: "%n%u"
         precision: 0
-        separator: ""
+        separator: "."
```

Comment:
Separator change

Reply:
Looks like its a comma is appropriate https://github.com/twitter/twitter-cldr-rb/blob/2b65e4b494924f80a8d15fc0fd2efd0cd8090ba0/resources/locales/ja/numbers.yml#L43